src/Dev/Tasks/MigrateFileTask.php

```diff
@@ -57,7 +57,13 @@ class MigrateFileTask extends BuildTask
 
     public function __construct()
     {
-        Deprecation::notice('4.12.0', 'Will be removed without equivalent functionality to replace it', Deprecation::SCOPE_CLASS);
+        Deprecation::withNoReplacement(function () {
```

Comment:
See https://github.com/silverstripe/silverstripe-admin/pull/1453#discussion_r1098134905 for this and any other similar changes in this PR